lib/parser.dart

```diff
@@ -485,7 +485,11 @@ class _Parser {
    *  mixin:              '@mixin name [(args,...)] '{' declarations/ruleset '}'
    *  include:            '@include name [(@arg,@arg1)]
    *                      '@include name [(@arg...)]
-   *  content             '@content'
+   *  content:            '@content'
+   *  -moz-document:      '@-moz-document' [ <url> | url-prefix(<string>) |
+   *                          domain(<string>) | regexp(<string) ]# '{'
+   *                        declarations
+   *                      '}'
```

Comment:
If it shows up again in a W3 draft, or another browser starts supporting it, we can make the `DocumentDirective` more generic with difference vendor prefixes.